conf/messages

```diff
@@ -493,6 +493,60 @@ otherBeneficiary.whenRemoved.error.invalid = Enter a real date when the benefici
 otherBeneficiary.whenRemoved.error.future = The date the beneficiary was removed from the trust must be today or in the past
 otherBeneficiary.whenRemoved.error.past = The date the beneficiary was removed must be on or after the date they were added
 
+otherBeneficiary.description.title = What is the description of the other type of beneficiary?
+otherBeneficiary.description.heading = What is the description of the other type of beneficiary?
+otherBeneficiary.description.checkYourAnswersLabel = What is the description of the other type of beneficiary?
+otherBeneficiary.description.hint = This must be 56 characters or less. For example, future grandchildren and future great-grandchildren.
+otherBeneficiary.description.error.required = Enter the description the other type of beneficiary
+otherBeneficiary.description.error.invalidFormat = The description the other type of beneficiary must only include letters a to z, numbers, ampersands (&), apostrophes, commas, forward slashes, full stops, hyphens, round brackets and spaces
```

Comment:
The description of the beneficiary must only include letters a to z, numbers, ampersands (&), apostrophes, forward slashes, full stops, hyphens, round brackets and spaces